ui/tsconfig.json

```diff
@@ -10,17 +10,14 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
```

Comment:
defining `typeRoots` is unnecessary. It is part of the default value.
https://www.typescriptlang.org/tsconfig#typeRoots